lib/MusicBrainz/Server/Edit/Relationship/RelatedEntities.pm

```diff
@@ -7,7 +7,7 @@ use MusicBrainz::Server::Data::Utils qw( type_to_model );
 
 requires 'directly_related_entities';
```

Comment:
If following the above suggestion, this should cause it to error if consumers don't implement `link_type`.
```suggestion
requires 'directly_related_entities', 'link_type';
```